web/app/reset-success/page.tsx

```diff
@@ -5,7 +5,7 @@ import ResetBackground from "@/components/Authentication/ResetBackground";
 const Reset: React.FC = () => {
   return (
     <div>
-      <ResetBackground description="Reset Success" Card={ResetSuccess} />
+      <ResetBackground description="Reset Success" card={ResetSuccess} action={undefined} />
```

Comment:
I think it's best we pass a void function (until it's implemented) to the ResetBackground as a prop instead of undefined.